tsconfig.browser-test.json

```diff
@@ -21,7 +21,15 @@
     "newLine": "LF",
     "outDir": "./test-browser",
     "rootDir": "./test",
-    "useUnknownInCatchVariables": false
+    "useUnknownInCatchVariables": false,
+    "paths": {
+      "@msinternal/body-string-rest": [

```

Comment:
Note here I will ping the package path to ensure files in `samples-dev` could build successfully